### DISCUSSION TITLE
Sort For Date and Table Row Style

### DISCUSSION THREAD

**Discussion by edwinx29** [MuhammadRafeh/react-native-datatable-component#17] [Ideas]
Hello there, I used this library and I think it has many potentials.
I already customize this library for date sorting on my own project, but still not optimized because it's not dynamically sorted for some date format.
I hope I can contribute to this project after solving this problem. I think many people need this library.

**Comment by MuhammadRafeh** [comment]
@edwinx29 Yes sure please contribute and make this module more stronger.
